e2e/testsuite/tx.go

```diff
@@ -64,7 +64,7 @@ func (s *E2ETestSuite) BroadcastMessages(ctx context.Context, chain ibc.Chain, u
 	broadcastFunc := func() (sdk.TxResponse, error) {
 		return cosmos.BroadcastTx(ctx, broadcaster, user, msgs...)
 	}
-	if s.relayers.ContainsRelayer(s.T().Name(), user) {
+	if s.relayerWallets.ContainsRelayer(s.T().Name(), user) {
```

Comment:
just noticing that `s.T().Name()` is no longer the correct key for this, changing this will make the diff super ugly so I might do this in a follow up.